test/run/ch-fromhost.bats

```diff
@@ -29,26 +29,23 @@ fromhost_ls () {
     find "$1" -xdev -name '*sotest*' -ls
 }
 
-@test 'ch-fromhost (Debian)' {
+@test 'ch-fromhost (CentOS)' {
     scope standard
-    prerequisites_ok debian9
-    img=${ch_imgdir}/debian9
-
-    libpath=$(ch-fromhost --lib-path "$img")
-    echo "libpath: ${libpath}"
+    prerequisites_ok centos8
+    img=${ch_imgdir}/centos8
 
     # --file
     fromhost_clean "$img"
     ch-fromhost -v --file sotest/files_inferrable.txt "$img"
     fromhost_ls "$img"
     test -f "${img}/usr/bin/sotest"
-    test -f "${img}${libpath}/libsotest.so.1.0"
-    test -L "${img}${libpath}/libsotest.so.1"
+    test -f "${img}/lib64/bind9-export/libsotest.so.1.0"
+    test -L "${img}/lib64/bind9-export/libsotest.so.1"
```

Comment:
That's a really bizarre place to put it. I wonder if we should put a comment acknowledging that and explaining why.

Reply:
I completely agree. I feel that not much is gained by hardcoding a path versus using `${libpath}` except we get to track whatever package happens to include a search path that the linker checks first. Could you elaborate on why you think this is something we should test?

Reply:
I thought it would get `/usr/local/lib`. Maybe that is true now with my tidy? But maybe we should just respect `ch-fromhost --lib-path` (which I think is now `--_lib-path`?).

Reply:
Looks like post "tidy" it is still getting `..../bind9-export`. I will revert to `${libpath}`.